src/libs/karm-sys/proc.cpp

```diff
@@ -2,6 +2,12 @@
 
 namespace Karm::Sys {
 
+static Mime::Url _pwd{_Embed::pwd().take()};
+
+Mime::Url const& pwd() {
+    return _pwd;
+}
+
```

Comment:
Don't do that, just call the function

https://en.cppreference.com/w/cpp/language/siof

Reply:
hey thanks for the resource, i thought that the static variable would get initialized when the `.cpp` is being translated. 
so i thought that, if we are executing `pwd()`, necessarily `_pwd` had already been initialized

mainly i copied from https://github.com/odoo/paper-muncher/blob/main/src/libs/karm-sys/chan.cpp

also the `getcwd` is always making a copy so i thought calling it only once would be better... from the resource you linked i checked that we could use the 'The Construct on First Use Idiom' and place the static init inside the method. don't you like this idea?